jobs/scrape_archives.py

```diff
@@ -128,16 +133,18 @@ def archive_url(product, version=None, platform=None, locale=None, nightly=None)
     if locale != "en-US":
         return None
 
-    url = "{}{}/candidates/{}-candidates/".format(ARCHIVE_URL, product, version)
+    product_url = product if product != "fennec" else "mobile"
+
+    url = "{}{}/candidates/{}-candidates/".format(ARCHIVE_URL, product_url, version)
     try:
         build_folders, _ = await fetch_listing(session, url)
         latest_build_folder = sorted(build_folders)[-1]
 
         url += "{}{}/{}/".format(latest_build_folder, platform, locale)
         _, files = await fetch_listing(session, url)
 
-        metadata_file = "{}-{}.json".format(product, version)
-        json_file = [f_["name"] for f_ in files if f_["name"].endswith(metadata_file)][0]
+        re_metadata = re.compile("{}-{}(.*).json".format(product, version))
+        json_file = [f_["name"] for f_ in files if re_metadata.match(f_["name"])][0]
```

Comment:
If I am reading correctly, we're expecting `firefox-54.0a2.json` for desktop/Firefox but `fennec-54.0a2.multi.android-arm.json` for Android. And if I'm right, this logic is slightly different for JSON files than for archives? Maybe it would be good to centralize these expectations in some kind of filename schemae, or at least add a comment showing some examples of what we expect?